schema/gks-core/gks-core-source.yaml

```diff
@@ -188,26 +188,40 @@ $defs:
     description: >-
       A structured representation of a code for a defined concept in a terminology or code system.
     properties:
+      code:
+        $ref: '#/$defs/code'
+        description: >-
+          A symbol uniquely identifying the concept, as in a syntax defined by the code system.
+          CURIE format is preferred where possible (e.g. 'SO:0000704' is the CURIE form of the
+          Sequence Ontology code for 'gene'). See https://www.w3.org/TR/curie/.
       label:
         type: string
         description: The human-readable name for the coded concept, as defined by the code system.
       system:
         type: string
         description: >-
           The terminology/code system that defined the code. May be reported as a free-text name
```

Comment:
@mbrush , @ahwagner and I had a discussion regarding `system` when reviewing my draft PR in va-spec where I initially used free text label for this field. However, @ahwagner believes URI should be used to align with FHIR.

Tagging @larrybabb to follow too